src/main/java/lotto/view/LottoOutput.java

```diff
@@ -0,0 +1,27 @@
+package lotto.view;
+
+import lotto.domain.TotalEqualNumbers;
+import lotto.service.BuyLotto;
+
+public class LottoOutput {
+    public void displayBuyLottoNumbers(BuyLotto buyLotto) {
+        buyLotto.getLottos().getLottos().stream()
+                .forEach(numbers -> System.out.println(numbers.getNumbers()));
+    }
+
+    public void displayWinOfResult(TotalEqualNumbers totalEqualNumbers) {
+        System.out.println("당첨 통계");
+        System.out.println("---------");
+        System.out.println("3개 일치 (5,000원) - " + totalEqualNumbers.getEqualsNumberCount3() + "개");
```

Comment:
```suggestion
        System.out.println(totalEqualNumbers.getCount()+ "개 일치 ("+totalEqualNumbers.getMoney()+") - " + totalEqualNumbers.getEqualsNumberCount3() + "개");
        
```

`totalEqualNumbers.getEqualsNumberCount3()` 의 갯수와  `3개 일치 (5,000원)` 의 개념이 같이 움직인다고 보장하기 힘든 구조인것 같아요 🤔 

그렇다면, 비슷한 속성을 하나의 객체로 받아서, 전달한다면 좀 더 유지보수성 과 설계가 탄탄해 지지 않을까요?